docs/README-agent-install.md

```diff
@@ -601,11 +602,15 @@ systemctl start my_chef
 
 [Chef Provisioning](README-chef-provisioning.md) - Installing & Configuring chef-client using the Chef Provisioning tool
 
-## References
+## <a name="references">References
```

Comment:
Or just remove the `<a name="references">` altogether as Adam's email points out this should already have an auto anchor.